miner/worker.go

```diff
@@ -1167,7 +1169,10 @@ loop:
 			// Reorg notification data race between the transaction pool and miner, skip account =
 			log.Trace("Skipping account with hight nonce", "sender", from, "nonce", tx.Nonce())
 			txs.Pop()
-
+		case errors.As(err, &errL1):
```

Comment:
Can we do this?

```suggestion
		case errors.Is(err, vm.ErrL1RPCError):
```

Reply:
we can't use `vm.ErrL1RPCError` in `.is` method, becuase it's struct that implements error and wraps another error inside. `.is` can accept only instance of any error. For this situation where `vm.ErrL1RPCError` is wrapper on another err I thought that `.as` is more convinient